portal-api/src/v1/roles/user-has-sufficient-role.js

```diff
@@ -0,0 +1,23 @@
+const { UKEF_OPERATIONS } = require('./roles');
+
+const userHasRole = ({
+  user,
+  targetRole,
+}) => {
+  const { roles: userRoles } = user;
+  return userRoles.includes(targetRole);
+};
+
+const userHasSufficientRole = ({
+  user,
+  allowedNonAdminRoles,
+}) => {
+  if (userHasRole({ user, targetRole: UKEF_OPERATIONS })) {
+    return true;
+  }
```

Comment:
The naming "allowedNonAdminRole" will fit better when the "I can access any route in portal-api" is `admin` rather than `ukef_operations`